packages/core/package.json

```diff
@@ -3,32 +3,34 @@
   "version": "2.0.0-alpha.2",
   "description": "A simple React component capable of building HTML forms out of a JSON schema.",
   "scripts": {
-    "build:lib": "rimraf lib && cross-env NODE_ENV=production babel -d lib/ src/",
-    "build:dist": "rimraf dist && cross-env NODE_ENV=production webpack --config webpack.config.dist.js",
-    "cs-check": "prettier -l \"{playground,src,test}/**/*.js\"",
-    "cs-format": "prettier \"{playground,src,test}/**/*.js\" --write",
-    "dist": "npm run build:lib && npm run build:dist",
-    "lint": "eslint src test playground",
+    "build": "npm run build:cjs && npm run build:es && npm run build:es:lib",
+    "build:cjs": "NODE_ENV=production BABEL_ENV=cjs babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./dist/cjs",
+    "build:es": "NODE_ENV=production BABEL_ENV=es babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./dist/es",
+    "build:es:lib": "NODE_ENV=production BABEL_ENV=es babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./lib",
```

Comment:
The NODE_ENV stuff here seems to not work on windows

Reply:
please see my other comments relating to UMD builds here, as this currently won't build for me I cannot check but basically need output as per UMD (this is commonjs iffe and amd all in one)

i.e.. 
``` js
(function (root, factory) {
    if (typeof define === 'function' && define.amd) {
        // AMD. Register as an anonymous module.
        define(['exports', 'b'], factory);
    } else if (typeof exports === 'object' && typeof exports.nodeName !== 'string') {
        // CommonJS
        factory(exports, require('b'));
    } else {
        // Browser globals
        factory((root.commonJsStrict = {}), root.b);
    }
}(this, function (exports, b) {
    //use b in some fashion.

    // attach properties to the exports object to define
    // the exported module properties.
    exports.action = function () {};
}));
```

Reply:
this basically needs cross-env to be stuck in front in package.json's.. already there in other places.

Reply:
Please note that this change in package.json to build both commonjs and esm really breaks things for me, basically there was one output before which was an amd module i.e. in the dist folder which I would dump with all my other js files

Reply:
note this now dumps lots of small files and not a single minified / bundled file

Reply:
@saulzi can you share some example code that relies on the UMD build? That way I can make sure that the way it outputs will work with that code.

Reply:
> this basically needs cross-env to be stuck in front in package.json's.. already there in other places.

Done

Reply:
> @Saulzi can you share some example code that relies on the UMD build? That way I can make sure that the way it outputs will work with that code.

@epicfaace @chanceaclark 

https://github.com/Saulzi/react-jsonschema-form/tree/amd-example
Here is an example of using react-jsonschema-form via UMD. using all three legacy module loader types across node, and the browser.
 
I found a few things i.e. synthetic defaults etc with the current build...

Why build commonjs when UMD does it all for legacy module loaders.. essentially newer build tools such as rollup / webpack use ESM, node is stuck using commonjs and browser can essentually do amd (via steal / require / dojo / curl?), commonjs (stealjs), umd script tags and esm modules.. 

things would be so much better if react was built to esm module because then it would play nicer with rollup etc and would allow me to load it via script type module rather than relying on AMD.

I am glad this commit is now in merged as its definitely an improvement but still needs a little bit of tweaking, @epicfaace does this also resolve issues with CI i.e. is the playground preview now up to date?

Reply:
@saulzi
You said "I found a few things i.e. synthetic defaults etc with the current build..." -- did you mean that you are able to use the UMD version now or are there still issues?

Reply:
@epicfaace it does work but really when referencing this in the browser and via amd you shouldn't have to use .default... it is basically because of synthetic exports.